<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1131</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>225</width>
      <height>511</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>ASCII art settings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Font</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="fontComboBox">
         <item>
          <property name="text">
           <string>Courier</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Palette</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="paletteComboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Symbols horizontal (max 200)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spin_box_width">
         <property name="maximum">
          <number>200</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Symbols vertical (max 200)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spin_box_height">
         <property name="maximum">
          <number>200</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="autoSizesButton">
         <property name="text">
          <string>Auto sizes</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Proportional sized</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="proportionalCheckBox">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Symbols per pixels (scale)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_3"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Image edditing</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Contrast</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="contrastSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Sharpness</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="opasitySlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Brightness</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="brightnessSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Blur</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="blurSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetImageButton">
       <property name="text">
        <string>Reset image settings</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Display settings on image</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="displayCheckBox">
         <property name="text">
          <string/>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="ascii_output">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>451</width>
      <height>451</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>6</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="image_art">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>401</width>
      <height>451</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="convert_button">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>472</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="autoConvertCheckBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>480</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1131</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLoad">
    <property name="title">
     <string>Load</string>
    </property>
    <addaction name="actionLoadImage"/>
   </widget>
   <widget class="QMenu" name="menuSave">
    <property name="title">
     <string>Save</string>
    </property>
    <addaction name="actionASCIIArtAsTxt"/>
    <addaction name="actionCopypaletteAsString"/>
    <addaction name="actionCopyArtInBuffer"/>
   </widget>
   <widget class="QMenu" name="menupalettes">
    <property name="title">
     <string>Palette</string>
    </property>
    <addaction name="actionCreatepalette"/>
    <addaction name="actionBrowsepalette"/>
   </widget>
   <widget class="QMenu" name="menuMode">
    <property name="title">
     <string>Mode</string>
    </property>
    <addaction name="actionBrightnessMode"/>
    <addaction name="actionLineMode"/>
   </widget>
   <widget class="QMenu" name="menuEditImage">
    <property name="title">
     <string>Edit image</string>
    </property>
    <addaction name="actionInverting"/>
    <addaction name="actionCrop"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionDevelopmentInfo"/>
   </widget>
   <addaction name="menuLoad"/>
   <addaction name="menuSave"/>
   <addaction name="menupalettes"/>
   <addaction name="menuMode"/>
   <addaction name="menuEditImage"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoadImage">
   <property name="text">
    <string>Image</string>
   </property>
  </action>
  <action name="actionASCIIArtAsTxt">
   <property name="text">
    <string>ASCII art as .txt</string>
   </property>
  </action>
  <action name="actionBrightnessMode">
   <property name="text">
    <string>Brightness (defalt)</string>
   </property>
  </action>
  <action name="actionLineMode">
   <property name="text">
    <string>Line</string>
   </property>
  </action>
  <action name="actionInverting">
   <property name="text">
    <string>Inverting</string>
   </property>
  </action>
  <action name="actionDevelopmentInfo">
   <property name="text">
    <string>GitHub</string>
   </property>
  </action>
  <action name="actionHow_to_use">
   <property name="text">
    <string>How to use</string>
   </property>
  </action>
  <action name="actionFAQ">
   <property name="text">
    <string>FAQ</string>
   </property>
  </action>
  <action name="actionpaletteAsDb">
   <property name="text">
    <string>DB</string>
   </property>
  </action>
  <action name="actionCopypaletteAsString">
   <property name="text">
    <string>Copy palette as string</string>
   </property>
  </action>
  <action name="actionSavepaletteAsTxt">
   <property name="text">
    <string>palette as .txt</string>
   </property>
  </action>
  <action name="actionSet_palette">
   <property name="text">
    <string>Set palette</string>
   </property>
  </action>
  <action name="actionpaletteEditor">
   <property name="text">
    <string>palette editor</string>
   </property>
  </action>
  <action name="actionpalette">
   <property name="text">
    <string>palette</string>
   </property>
  </action>
  <action name="actionpalettesDb">
   <property name="text">
    <string>palettes db</string>
   </property>
  </action>
  <action name="actionHightlight">
   <property name="text">
    <string>Hightlight</string>
   </property>
  </action>
  <action name="actionCrop">
   <property name="text">
    <string>Crop</string>
   </property>
  </action>
  <action name="actionCreatepalette">
   <property name="text">
    <string>Create palette</string>
   </property>
  </action>
  <action name="actionBrowsepalette">
   <property name="text">
    <string>Browse palettes</string>
   </property>
  </action>
  <action name="actionMainSettings">
   <property name="text">
    <string>Theme</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="text">
    <string>Font</string>
   </property>
  </action>
  <action name="actionBrowse_palettes">
   <property name="text">
    <string>Browse palettes</string>
   </property>
  </action>
  <action name="actionCopyArtInBuffer">
   <property name="text">
    <string>Copy ASCII art in buffer</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
